pom.xml

```diff
@@ -174,43 +166,23 @@
   </dependencies>
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.222.x</artifactId>
+        <version>16</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
       <dependency>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>structs</artifactId>
-        <version>1.14</version>
       </dependency>
       <dependency>
         <groupId>org.jenkins-ci</groupId>
         <artifactId>symbol-annotation</artifactId>
-        <version>1.14</version>
-      </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>script-security</artifactId>
-        <version>1.44</version>
-        <scope>test</scope>
-      </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>scm-api</artifactId>
-        <version>2.2.7</version>
-        <scope>test</scope>
       </dependency>
+
     </dependencies>
   </dependencyManagement>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.jenkins-ci.tools</groupId>
-        <artifactId>maven-hpi-plugin</artifactId>
-        <extensions>true</extensions>
-        <configuration>
-          <pluginFirstClassLoader>true</pluginFirstClassLoader>
```

Comment:
Revert, this is critical.

Reply:
Why is it critical (so I can put a comment above that it should not be touched) ?
Also, shouldn't below be ${java.level} instead of 1.5?

Reply:
> Why is it critical

Because of one of the libraries it is using, SSHD IIRC. Without `pluginFirstClassLoader` that mode of the plugin will just throw errors. https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/

`compatibleSinceVersion` should also stay, though at this point it is mostly of historical interest since it is unlikely people would be updating since such an old version.